test/parallel/test-debug-usage.js

```diff
@@ -6,9 +6,8 @@ const spawn = require('child_process').spawn;
 const child = spawn(process.execPath, ['debug']);
 child.stderr.setEncoding('utf8');
 
-const expectedUsageMessage = `Usage: node debug script.js
-       node debug <host>:<port>
-       node debug -p <pid>
+const expectedUsageMessage = `Usage: node-inspect script.js
```

Comment:
Note to future self: This test will need to be updated once we pull in @addaleax's usage text fix. https://github.com/nodejs/node-inspect/pull/20/files